.github/workflows/core-lint-fmt.yml

```diff
@@ -162,11 +162,6 @@ jobs:
 
       - name: Get current date
         run: echo "CURRENT_DATE=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
-        if: matrix.platform == 'macos-latest' || matrix.platform == 'ubuntu-latest'
-
```

Comment:
Time is needed for the cache.

Reply:
This `if` condition is always false. Check Actions result.